npre/umbral.py

```diff
@@ -12,7 +12,28 @@
 
 
 EncryptedKey = namedtuple('EncryptedKey', ['ekey', 're_id'])
-RekeyFrag = namedtuple('RekeyFrag', ['id', 'key'])
+
+
+class RekeyFrag(object):
+
+    DELIMETER = b"||"
```

Comment:
Why a delimiter at all if they have the same size?
Besides, with a delimiter you'll have a very substantial probability of this delimiter appearing somewhere else randomly

Reply:
Yeah, I was almost certain that this was the wrong way to proceed when I wrote it.  I wasn't 100% sure if they were the right size, or if they were, if that was the right way.